specification/synapse/resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/bigDataPool.json

```diff
@@ -437,6 +437,13 @@
           "$ref": "#/definitions/LibraryRequirements",
           "description": "Library version requirements"
         },
+        "customLibraries": {
+          "description": "List of custom libraries/packages associated with the spark pool.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/LibraryInfo"
+          }
+        },
```

Comment:
It's a good idea to include a nextLink property and annotate the schema with x-ms-pageable.

Reply:
This is one of the attributes in response object(BigDataPoolResourceProperties) and doesn't support nextlink. 
Next link is only supported by BigDataPoolResourceInfoListResult only in case of list api